posthog/utils_cors.py

```diff
@@ -39,4 +39,5 @@ def cors_response(request, response):
     response["Access-Control-Allow-Headers"] = "X-Requested-With,Content-Type" + (
         "," + ",".join(allow_headers) if len(allow_headers) > 0 else ""
     )
+    response["Vary"] = "Origin"
```

Comment:
```suggestion
    response["Vary"] = "Accept-Encoding, Origin"
```

i think we need that in case we have folk requesting brotli or something... if nobody is then it doesn't matter, if somebody is then it really does